src/jshint.js

```diff
@@ -653,12 +653,15 @@ var JSHINT = (function() {
           case "3":
           case "5":
           case "6":
+          case "7":
```

Comment:
hmmm --- maybe for some kind of design aesthetic, should save the esversion option as it was written, and print it rather than "7" when converting it to text. That way, there won't be a mismatch if `esversion=2015`